packages/frontend/src/components/Home/icon/boltIcon.tsx

```diff
@@ -0,0 +1,29 @@
+const BoltIcon = () => (
+    <svg
```

Comment:
hey @Ifunanyacollins is there chance that we can use unicode character for bolt? ⚡️

Reply:
Ah! I thought about Unicode, but Safari still renders bolt Unicode differently. Since the goal is to have consistent icons across all browsers, it's best that we use SVGs here.

what are your concerns about using SVG?

Reply:
Hi @IrakliJani, I figured out how to make the Unicode work.
What do you think about this?
<img width="1287" alt="Screenshot 2023-02-13 at 18 55 28" src="https://user-images.githubusercontent.com/31220512/218535401-ba2adab8-9b8a-4889-8838-ada6a496a49d.png">
<img width="897" alt="Screenshot 2023-02-13 at 18 57 19" src="https://user-images.githubusercontent.com/31220512/218535734-b152d21a-1a22-46bd-828f-eca778949ef7.png">